src/frontend/main.js

```diff
@@ -87,12 +90,23 @@ $(function() {
     };
 
     const removeBookmark = function() {
-        const bookmark = results.find('.selected');
-        const id = bookmark.data('id');
-        bookmarksCollection.remove(id, function() {
-            selectNext();
-            bookmark.remove();
-            loadBookmarks();
+        sweetAlert({
```

Comment:
Totally, you should move it to module, set up settings there and expose some minimal api to call it here